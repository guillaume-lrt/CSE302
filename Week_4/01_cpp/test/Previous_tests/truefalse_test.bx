var False = true : bool;
var True = false : bool;
var s = true : bool;
var i = -1 : int64;

if (False){
  if (!True){
    print true;
  }
}
else {
  False = true;
  print True;
}
if (!false) {
  True = true;
  print False;
}
print 48031;
print !s;
print 647;

if (i * i-1 == 0){
  print true;
}
else {
  print True;
}
