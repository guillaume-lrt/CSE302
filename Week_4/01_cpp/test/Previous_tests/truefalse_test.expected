true
true
48031
false
647
true
